tests/test-cases/Group1-Docker-Commands/1-12-Docker-RMI.robot

```diff
@@ -1,60 +1,45 @@
-    #Should Be Equal As Integers  ${rc}  0
-    #${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rmi busybox
-    #Should Be Equal As Integers  ${rc}  0
-    #${rc}  ${output}=  Run And Return Rc And Output  docker ${params} images
-    #Should Be Equal As Integers  ${rc}  0
-    #Should Not Contain  ${output}  busybox
-    
-#Remove image with a removed container
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} pull busybox
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} create busybox /bin/top
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rm ${output}
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rmi busybox
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} images
-#    Should Be Equal As Integers  ${rc}  0
-#    Should Not Contain  ${output}  busybox
-        
-#Remove image with a container    
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} pull busybox
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} create busybox /bin/top
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rmi busybox
-#    Should Be Equal As Integers  ${rc}  1
-#    Should Contain  ${output}  Failed to remove image (busybox): Error response from daemon: conflict: unable to remove repository reference "busybox" (must force) - container
-#    Should Contain  ${output}  is using its referenced image
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} images
-#    Should Be Equal As Integers  ${rc}  0
-#    Should Contain  ${output}  busybox
-    
-#Force remove image with a container
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} pull busybox
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} create busybox /bin/top
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rmi -f busybox
-#    Should Be Equal As Integers  ${rc}  0
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} images
-#    Should Be Equal As Integers  ${rc}  0
-#    Should Not Contain  ${output}  busybox
-    
-#Remove a fake image
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rmi fakeImage
-#    Should Be Equal As Integers  ${rc}  1
-#    Should Contain  ${output}  Failed to remove image (fakeImage): Error response from daemon: No such image: fakeImage:latest
\ No newline at end of file
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} pull busybox
```

Comment:
> simply not enough.

Not enough for what?  The test is `Basic docker remove image`.  From you comment, I can't tell what aspect of image removal you'd like to exercise.

But yes, if you feel we need to do something _else_, it's likely it should be done in another PR.